CAIPs/caip-104.md

```diff
@@ -0,0 +1,229 @@
+---
+caip: 104
+title: Namespace Reference Purpose and Guidelines
+status: Active
+type: Meta
+author: Juan Caballero (@bumblefudge), Wayne Chang (@wyc)
+created: 2022-03-27
+---
+
+# What is a Namespace Reference?
+
+Previously, the specifics of blockchain ecosystems and namespaces were defined
+in single stand-alone CAIPs alongside cross-chain CAIPs. Now, as the system has
+matured and expanded, it has become clear that namespaces are more complex and
+multi-dimensional than can be efficiently governed over time in the current CAIP
+single-document structure; instead, a Namespace Reference is proposed that
+splits out the application of each cross-chain CAIP to a given namespace as a
+distinct document with its own ratification/supersession lifecycle.  This way,
+long-term specifications for low-level primitives like Chain IDs and addresses
+can be defined once and not be superseded by the addition or modification of
+higher-level specifications for RPC interfaces or multiple asset types.
+
+In practical terms, that means each blockchain ecosystem's namespace is defined
+by a folder containing one document per applicable CAIP.  At a high-level, the
```

Comment:
when reading this I would ask myself "why not just a document if it is *one* document per folder.
Having attended the last call I know there will be more documents in this folder - maybe we should reflect this here to avoid WTF moments in humans reading this.

Reply:
one document **per caip**, meaning many documents.  I reworded it to be clearer, thanks!